src/Mod/PartDesign/Gui/TaskHelixParameters.h

```diff
@@ -82,7 +82,8 @@ private Q_SLOTS:
     void changeEvent(QEvent *e) override;
     bool updateView() const;
     void getReferenceAxis(App::DocumentObject *&obj, std::vector<std::string> &sub) const;
-
+    void startReferenceSelection(App::DocumentObject* profile, App::DocumentObject* /*base*/);
```

Comment:
@davidosterberg Can you please add override at the end as otherwise clang will report several warnings?

Reply:
done